ui/pages/routes/routes.component.js

```diff
@@ -642,7 +642,7 @@ export default class Routes extends Component {
         })}
         dir={textDirection}
       >
-        {shouldShowNetworkDeprecationWarning && <DeprecatedTestNetworks />}
+        {shouldShowNetworkDeprecationWarning ? <DeprecatedNetworks /> : null}
```

Comment:
we can use original syntax?

Reply:
I think we can avoid `&&` and use ternary operators here re: https://github.com/MetaMask/metamask-extension/pull/21933#discussion_r1402468795